docs/react-testing-library/migrate-from-enzyme.mdx

```diff
@@ -339,30 +338,27 @@ test('handles click correctly', () => {
 })
 ```
 
-Nice! With the help of other modules provided by the Testing Library, we can
-test our components smoothly! To learn more about the `user-event` library, you
-can have a look at its
-[GitHub repository](https://github.com/testing-library/user-event#table-of-contents)
+Nice!
 
 ### Triggering class methods in tests (`wrapper.instance()`)
 
-As we already discussed, we are against testing implementation details and
-things that users are not aware of, and we aim to test and interact with the
-component like how our users would.
+As we already discussed, we recommend against testing implementation details and
+things that users will not be aware of. We aim to test and interact with the
+component more like how our users would.
 
-> if your test uses instance() or state(), know that you're testing things that
-> the user couldn't possibly know about or even care about, which will take your
-> tests further from giving you confidence that things will work when your user
-> uses them.
+> If your test uses \`instance()\` or \`state()\`, know that you're testing
+> things that the user couldn't possibly know about or even care about, which
+> will take your tests further from giving you confidence that things will work
```

Comment:
```suggestion
> takes your tests further from giving you confidence that things work
```

Reply:
This is a direct quote from the link below, so I wouldn't want to change it even if it reads better.